Sources/Commands/SwiftBuildTool.swift

```diff
@@ -303,7 +303,7 @@ enum CleanMode: CustomStringConvertible {
         case "dist"?, "distribution"?:
             self = .dist
         default:
-            throw OptionParserError.invalidUsage("invalid clean mode: \(rawValue)")
+            throw OptionParserError.invalidUsage("invalid clean mode: \(rawValue!)")
```

Comment:
wouldn't `rawValue ?? "nil"` be better in case someone ends up removing `nil` from switch case